vendor/github.com/moby/swarmkit/v2/manager/controlapi/service.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/docker/distribution/reference"
+	"github.com/distribution/reference"
```

Comment:
I'll bring this one in; this change was the most relevant change in this update (getting rid of more uses of the deprecated package)